### PR TITLE
React based configuration creation support for 8.3

### PR DESCRIPTION
**Pull request by justnoxx**

### CODE REVIEW

File: pages/EC-WebSphere_help.xml [hunk 1]
```diff
@@ -2733,6 +2733,10 @@
                 <img src="../../plugins/EC-WebSphere/images/ModifyApplicationClassLoader/EC-WebSphereModifyApplicationClassLoader4.png" />
             </div>
             <h1 id="rns">Release Notes</h1>
+            <h2>@PLUGIN_KEY@ 2.3.1</h2>
+            <ul>
+                <li>Internal improvements for 8.3 release.</li>
```
Comment (urvashisingh) [inline pages/EC-WebSphere_help.xml]: This release note is too vague. Instead, lets add these depending on the change being done. The actual notes to add to the release notes section are the parts in quotes.
For sub-stories of CEV-17708 -> "Configured the plugin to allow the ElectricFlow UI to render the plugin procedure parameters entirely using the configured form XMLs."
For sub-stories of CEV-17558 ->  "Enabled the plugin for managing the plugin configurations in-line when defining an application process step or a pipeline stage task."

File: src/main/resources/project/project.xml [hunk 2]
```diff
@@ -6,6 +6,83 @@
...
     <workspaceName />
     <propertySheet>
+        <!-- CEV-17558 improvements -->
+        <property>
+            <propertyName>ec_config</propertyName>
+            <description/>
+            <propertySheet>
+                <!-- WebSphere doesn't have custom GWT logic. Setting it for plugin context -->
+                <property>
+                    <propertyName>ec_formXmlCompliant</propertyName>
```
Comment (urvashisingh) [inline src/main/resources/project/project.xml]: ec_formXmlCompliant property should be at the plugin level *not* within ec_config property sheet.

File: src/main/resources/project/project.xml [hunk 3]
```diff
@@ -6,6 +6,83 @@
...
+                    <value>true</value>
+                </property>
+                <property>
+                    <propertyName>configLocation</propertyName>
+                    <expandable>0</expandable>
+                    <value>websphere_cfgs</value>
+                </property>
+                <property>
+                    <propertyName>fields</propertyName>
```
Comment (urvashisingh) [inline src/main/resources/project/project.xml]: Lets keep the list of fields to display in the list page limited to 2-3 including the configuration name which is the default.
So, for websphere, just add the websphere url here. This is also consistent with the fields we are currently showing for the plugin in the plugin manager page.

File: src/main/resources/project/project.xml (hunk 2 above)
Comment (justnoxx) [reply src/main/resources/project/project.xml]: Thanks, will fix it

**Pull request merged**